src/components/text-editor/text-editor.spec.tsx

```diff
@@ -1151,4 +1114,49 @@ describe("TextEditor", () => {
     // Clear Mock
     (window.scrollTo as jest.Mock).mockRestore();
   });
+
+  /* These next two tests can be removed when we remove the focusRedesignOptOut prop */
+  it("does not apply focus styling when isFocused is true and focusRedesignOptOut is true", () => {
```

Comment:
**question**: I've no objected to adding these tests in but it seems against the decision that has been taken elsewhere, which is to ignore all "old focus styles" in unit tests and test them in Cypress instead? I'm OK with either approach but we should be consistent.

Reply:
Specifically on this one, the ignore wasn't being applied properly. We had a look into it and Istanbul wasn't showing which branches weren't covered, so we figured rather than wasting time we would have to stick a test in for this exceptional case, unfortunately.

Reply:
weird, but OK - no harm done. Thanks for the explanation.